lib/spack/spack/installer.py

```diff
@@ -412,6 +413,20 @@ def clear_failures():
     spack.store.db.clear_all_failures()
 
 
+def combine_phase_logs(pkg):
+    """
+    Each phase will produce it's own log, so this function aims to cat all the
+    separate phase log output files into the pkg.log_path.
+    Args:
+        pkg (Package): the package that was built and installed
+    """
+    log_files = pkg.phase_log_files
+
+    with open(pkg.log_path, 'w') as fd:
```

Comment:
`fd` is a bit of a weird variable name here as it's typically used for file descriptors (low-level ints returned by `os.open()`).  I would use `log_file` or something here, as this is really a file object.

Reply:
Will do. I always uesd "filey" when I was new to Python, but then always say "fd" for file descriptor, and fh for file handle, and thought that was best practice.